app/lib/theme/widgets/SelectRfpWidget.tsx

```diff
@@ -0,0 +1,81 @@
+import { WidgetProps } from "@rjsf/core";
```

Comment:
This is a specific widget for the application, not part of the theme (which will eventually be complete enough to be used in conjunction with the button service development toolkit in any project).
I think we should put it next to the ProjectForm file and add it to the list of widgets available directly to the form ([example](https://github.com/bcgov/cas-ciip-portal/blob/8eb4a75b695b1fb489d5acc51cb0b52d8e8eaa4c/app/containers/Facilities/AddFacilityModal.tsx#L98) in ciip)

Reply:
I moved the application specific component into the Form directory and added it to the 'widgets' object on the Form